sympy/stats/rv_interface.py

```diff
@@ -208,6 +208,31 @@ def skewness(X, condition=None, **kwargs):
     """
     return smoment(X, 3, condition, **kwargs)
 
+def kurtosis(X, condition=None, **kwargs):
+    """
+    Measure of the tailedness(outliers) of the probability distribution
+
+    Kurtosis of any univariate normal distribution is 3. Kurtosis less than
+    3 means that the distribution produces fewer and less extreme outliers
+    than the normal distribution
+
+    kurtosis(X) = E( ((X - E(X))/sigma)**4 )
+
+    Examples
+    ========
+
+    >>> from sympy.stats import kurtosis, Exponential, Normal
+    >>> from sympy import Symbol
+    >>> X = Normal('X', 0, 1)
+    >>> kurtosis(X)
+    3
+    >>> rate = Symbol('lambda', positive=True, real=True, finite=True)
+    >>> Y = Exponential('Y', rate)
+    >>> kurtosis(Y)
+    9
+    """
+    return smoment(X, 4, condition, **kwargs)
```

Comment:
Probably you would like to pass `condition` as an optional argument, so `condition=condition`.

Reply:
Can I also make the same change for skewness function?

Reply:
Yes, that would make sense.